### ISSUE TITLE
chai should issues

### ISSUE THREAD

**Issue by beckend** [speedskater/babel-plugin-rewire#140]
```javascript
import imageResolver from './img-resolver';

const should = chai.should();
describe('ImageResolver', () => {
  it('should throw an error using on browser', () => {
    try {
      imageResolver();
    } catch (error) {
      should.exist(error);
    }
  });
});
```
Object is not a constructor (evaluating '_get__('should').exist(error)')

It works fine without rewire.

**Comment by StyMaar** [comment]
it looks like a duplicate of #115 

As a work-around you can try to name your constant `_should` instead of `should`. This should fix the issue until #115 is solved.

**Comment by speedskater** [comment]
Fixed the issue in course of fixing #115. Added test which explicitly checks support of chai should.
